homeassistant/components/sleepiq/entity.py

```diff
@@ -33,6 +35,18 @@ def __init__(
         self._attr_name = f"SleepNumber {bed.name} {sleeper.name} {SENSOR_TYPES[name]}"
         self._attr_unique_id = f"{bed.id}_{sleeper.name}_{name}"
 
+    @property
+    def device_info(self) -> DeviceInfo:
+        """Return information about the device."""
+
+        return DeviceInfo(
+            identifiers={(DOMAIN, self.bed.id)},
+            connections={(device_registry.CONNECTION_NETWORK_MAC, self.bed.mac_addr)},
+            manufacturer="SleepNumber",
+            name=self.bed.name,
+            model=self.bed.model,
```

Comment:
```suggestion
            connections={(device_registry.CONNECTION_NETWORK_MAC, bed.mac_addr)},
            manufacturer="SleepNumber",
            name=bed.name,
            model=bed.model,
```
We usually don't set `identifiers` when we have `connections` unless you need it for `via_device`